### PR TITLE
created task_edit form and page which is connected to tasks page

### PR DESCRIPTION
**Pull request by fbbunny**
Basic edit page is created but needs to be designed nicer which could be next PR; 
any suggestions on the best practice in terms of form design instead of making a widget for each line?

### CODE REVIEW

File: web/templates/pages/tasks.html [hunk 1]
```diff
@@ -30,6 +30,9 @@
 								none
 							{% endif %}
 						</p>
+						<div class="col-lg-6 col-sm-7">
+			<a class="btn btn-default" href="{% url 'task-edit' task_id=task.id %}"><span class="glyphicon glyphicon-pencil"></span></a>
```
Comment (gandalfar) [inline web/templates/pages/tasks.html]: it looks like you're missing a few indents here

File: web/templates/pages/task_edit.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			<button type="submit" class="save btn btn-default">Save</button>
+			</form>
+		</div>
+	{% endif %}	
+
+{% endblock content %}
+
+{% block custom_js %}
+	<script type="text/javascript" src="{% static 'js/jquery.datetimepicker.js' %}"></script>
+{% endblock custom_js %}
```
Comment (gandalfar) [inline web/templates/pages/task_edit.html]: missing newline at the end :)

**Comment by ialja** [comment]
@livike would be a good reviewer for this PR :)

File: web/templates/pages/tasks.html [hunk 3]
```diff
@@ -30,6 +30,9 @@
 								none
 							{% endif %}
 						</p>
+						<div class="col-lg-6 col-sm-7">
```
Comment (livike) [inline web/templates/pages/tasks.html]: If you use to define col classes use the enclosing "row" class div

File: web/templates/pages/task_edit.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block custom_css %}
+	<link rel="stylesheet" type="text/css" href="{% static 'css/jquery.datetimepicker.css' %}" >
```
Comment (livike) [inline web/templates/pages/task_edit.html]: You will need the datetimepicker files enclosed when you define the appropriate id and class using widgets in your form definition. But for now you can leave here.

**Comment by livike** [comment]
The edit task form is working. 
At this moment I would suggest to add login authentication check on the template and on the view (see my recent PR #191)
In a separate PR enhance the form definition by defining widgets, labels, help texts, error texts... and on the template define the form elements manually.
Pay attention to the datetimepicker usage (css and javascript)

File: web/templates/pages/task_edit.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+			{{ taskform.as_p }}
+			<button type="submit" class="save btn btn-default">Save</button>
+			</form>
+		</div>
+	{% endif %}	
+
+{% endblock content %}
+
+{% block custom_js %}
+	<script type="text/javascript" src="{% static 'js/jquery.datetimepicker.js' %}"></script>
```
Comment (livike) [inline web/templates/pages/task_edit.html]: The datetimepicker script is now enclosed in the base.html template, so you don't need the entire custom_js block

**Comment by ialja** [comment]
@fbbunny Can you please check the comments (esp. regarding the datetimepicker) & sync your fork, so we can merge this afterwards?

**Comment by gandalfar** [comment]
:+1:

**Pull request merged**